labellab-flask/api/controllers/teamscontroller.py

```diff
@@ -0,0 +1,329 @@
+    @jwt_required
+    def get(self):
+        """Handle GET request for this view. Url ---> /api/v1/team/get"""
+        current_user = get_jwt_identity()
+        
+        try:
+            data = get_data(current_user)
+
+            if not data:
+                response = {
+                    "success": False,
+                    "msg": "Data could not be fetched"}
+                return make_response(jsonify(response)), 404
+
+            all_teams = data["all_teams"]
+
+            if not all_teams:
+                response = {
+                    "success": False,
+                    "msg": "Data could not be fetched"
+                    }
+                return make_response(jsonify(response)), 404
+
+            response = {
+            "success": True,
+            "msg": "Projects fetched successfully.",
+            "body": all_teams
+            }
+
+            return make_response(jsonify(response)), 200
+
+        except Exception as err:
+            print("Error occured: ", err)
+            response = {
+                "success": False,
+                "msg": "Data could not be fetched."
+                }
+            return make_response(jsonify(response)), 500
+
+class TeamInfo(MethodView):
+    """This class handles deletion, updating and fetching a team."""
+    @jwt_required
+    def get(self, project_id, team_id):
+        """Handle GET request for this view. Url --> /api/v1/team/team_info/<int:project_id>/<int:team_id>"""
+        try:
+            if not team_id:
+                response = {
+                    "success": False,
+                    "msg": "Provide all the required data."
+                }
+                return make_response(jsonify(response)), 400
+            team = find_by_id(team_id)
+            if not team:
+                response = {
+                    "success": False,
+                    "msg": "Team not found."
+                }
+                return make_response(jsonify(response)), 404
+            
+            response = {
+                "success": True,
+                "msg": "Team found.",
+                "body": team
+            }
+            return make_response(jsonify(response)), 200
+        except Exception:
+            response = {
+                "success": False,
+                "msg": "Something went wrong!!"
+            }
+            return make_response(jsonify(response)), 500
+    
+    @jwt_required
+    def delete(self, project_id, team_id):
+        """Handle DELETE request for this view. Url --> /api/v1/team/team_info/<int:project_id>/<int:team_id>"""
+        current_user = get_jwt_identity
+        roles = get_user_roles(current_user, project_id)
+        
+        if not roles.index("admin"):
+            print("Error occured: user not admin")
+            response = {
+                    "success": False,
+                    "msg": "User not admin."
+                }
+            return make_response(jsonify(response)), 400
```

Comment:
User is not the admin. so he cannot delete it. so status should should be 403 right?

Reply:
> Fix the status codes then run and check your code, please. 🙂

Sorry these errors may have arise when resolving conflicts. Will fix them right away !